### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by ArPimentel**

### CODE REVIEW

**Review by td6904 — APPROVED** [review]

File: src/app/about/about.component.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
```
Comment (Benja-san) [inline src/app/about/about.component.html:1]: remove all doctypes since it is already included inside the html entry file

File: src/app/about/about.component.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+<html>
+  <body>
+    <div class="about-card">
```
Comment (Benja-san) [inline src/app/about/about.component.html:4]: fix all issues I commented on your PR allRecipes

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: src/app/about/about.component.html (hunk 2 above)
Comment (ArPimentel) [reply src/app/about/about.component.html:4]: It's done

File: src/app/about/about.component.html (hunk 1 above)
Comment (ArPimentel) [reply src/app/about/about.component.html:1]: It's done

**Review by td6904 — DISMISSED** [review]

**Review by td6904 — APPROVED** [review]

File: src/app/about/about.component.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
+<html>
```
Comment (Benja-san) [inline src/app/about/about.component.html:1]: No html tag or body please, it is already here in your entry global file

**Review by Benja-san — CHANGES_REQUESTED** [review]

**Review by td6904 — APPROVED** [review]

File: src/app/footer/footer.component.scss [hunk 4]
```diff
@@ -0,0 +1,62 @@
+* {
```
Comment (Benja-san) [inline src/app/footer/footer.component.scss:1]: why is that here, it is supposed to be inside the global scss file. I accept it only once then you refactor it for next sprint

**Review by Benja-san — APPROVED** [review]
Read comment before merging
